ssn/rdf/ontology/core/sosa-common.ttl

```diff
@@ -229,7 +229,14 @@ sosa:Execution
   a owl:Class ;
   rdfs:label "Execution of a Procedure"@en ;
   skos:definition """
-  Act of carrying out a Procedure. This class generalizes the Observation, Actuation, and Sampling classes. It is not expected to be used directly.
+  Act of carrying out a Procedure. 
+  """@en ;
+  skos:scopeNote """
+  Execution generalizes the Actuation, Observation, and Sampling classes. 
+  It appears in the ontology to support the description of a common pattern for executions and their properties. 
+
+  Execution may be considered to be abstract, so individuals SHOULD NOT be declared to be members of only this class. 
+  Applications SHOULD use members of concrete sub-classes of Execution.
```

Comment:
I don't understand where 'object properties' would appear in the text. 

I used 'members' in the sense of the 'class extension'. 
Every individual of type `ex:Class1` is a _member_ of that class. 

I agree that the two SHOULD sentences are redundant. 
Perhaps it would be better phrased as 
> Execution may be considered to be abstract, so any individual Execution SHOULD be declared to be a member of one of the concrete sub-classes, and not of the Execution class itself.